pyinstaller.py

```diff
@@ -139,10 +139,6 @@ def pyinstall(source_folder):
                     % (command, source_folder, conan_server_path, win_ver),
                     cwd=pyinstaller_path, shell=True)
 
-    conan_bin = os.path.join(pyinstaller_path, 'dist', 'conan')
```

Comment:
I think we want to remove the full ``subprocess.call`` above and the ``_run_bin()`` below?

Reply:
let's remove them!